### DISCUSSION TITLE
Basic UI framework for SeaweedFS

### DISCUSSION THREAD

**Discussion by chrislusf** [seaweedfs/seaweedfs#3706] [Ideas]
Planning to create a separate UI server for SeaweedFS. Need help on ideas and suggestions, or maybe just create a prototype.

Business Requirements:
* Manage master, volume servers, filers, and S3

Technical Requirements:
* Minimum UI style. Better use some off-the-shelf UI theme.
* Go (Gin?) server
* React (because it seems simpler and easier to iterate)

**Comment by ryanrussell** [comment]
Hey Chris, 

I'm more or less doing this anyways for my own use-case, although it definitely will not be in react :)

Do you have a messenger or something to chat and iterate more quickly? May be able to help on this... Sending a private invite.

-r

**Reply by chrislusf** [reply]
I am on slack. The link is on the read me page.

**Comment by gaby** [comment]
@chrislusf Have you consider https://github.com/gofiber/fiber for the backend.

**Reply by chrislusf** [reply]
It's for an admin ui and needs to be easy to maintain. Performance is not critical.
